sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -261,6 +264,15 @@ def test_pardo_state_only(self):
     index_state_spec = userstate.CombiningValueStateSpec(
         'index', beam.coders.VarIntCoder(), sum)
 
+    # We want to use type hints which do not work with the Create transform
+    class SimpleCreate(beam.DoFn):
```

Comment:
@robertwb are you planning to fix `Create` so we won't need this workaround?

Reply:
I just saw that there is #6749.

Reply:
Removed this now that #6749 is merged.

Reply:
I think we still need the fix, portable python tests were failing, re-added it again.